react/src/lib/MenuOverlay/index.js

```diff
@@ -1,17 +1,31 @@
 /** @component menu */
 
 import React from 'react';
+import { findDOMNode } from 'react-dom';
```

Comment:
`findDOMNode` has ben deprecated in StrictMode. What are the ramifications of using it here? Are we ok with [this](https://reactjs.org/docs/strict-mode.html#warning-about-deprecated-finddomnode-usage) caveat?